src/models/generation.jl

```diff
@@ -13,3 +13,6 @@ function IS.get_limits(valid_range::Union{NamedTuple{(:min,:max)}, NamedTuple{(:
     return (min = valid_range.min, max = valid_range.max, zero = 0.0)
 end
 
+get_rating(gen::Generator) = get_rating(get_tech(gen))
```

Comment:
***This comment is not about the PR***

A get rating for branches is missing here. @claytonpbarrows what should we do about MonitoredLines